app/helpers/questionnaire_responses_helper.rb

```diff
@@ -0,0 +1,16 @@
+module QuestionnaireResponsesHelper
+  def latest_active_questionnaire_id(type)
+    Questionnaire.active.where(questionnaire_type: type).order(dsl_version: :desc).first.id
+  end
+
+  def month_date_str
+    @month_date.strftime("%Y-%m-%d")
```

Comment:
A QuestionnaireResponse service has an instance variable `@month_date` based on which the string is generated. We're using that in the helper module.
It is not considered best practice to use instance variables in a module, but is [acceptable in the rails world](https://stackoverflow.com/questions/70355343/ruby-on-rails-helper-method-instance-variable), and we're using it in other modules in Simple too.